kobo/apps/accounts/mfa/views.py

```diff
@@ -66,3 +70,15 @@ class MfaListUserMethodsView(ListAPIView):
     def get_queryset(self) -> QuerySet:
         mfa_model = get_mfa_model()
         return mfa_model.objects.filter(user_id=self.request.user.id)
+
+
+class IsMfaEnabled(BasePermission):
+    def has_permission(self, request, view):
+        mfa_available = MfaAvailableToUser.objects.filter(
+            user=get_database_user(request.user)
+        ).exists()
+        return mfa_available
```

Comment:
Let's try to be consistent with main app (`kpi`). Please the code in its own file `permissions.py` at the sub-app level.

Reply:
ok,